### PR TITLE
2024 Workshop page

### PR DESCRIPTION
**Pull request by tzanio**
Initial draft for the 2024 MFEM community workshop page.

### CODE REVIEW

File: src/workshop.md [hunk 1]
```diff
@@ -49,26 +57,9 @@ MFEM workshop.
...
-| 2:20-2:40 | **Break** | Discussions on [Slack](https://mfemworkshop.slack.com) |
-| 2:40-3:00 | **Wrap-up & Contest Winners** ([PDF](pdf/workshop23/21_Fisher_Wrap_Up.pdf), [video](https://youtu.be/DfqH7n1Sekg)) | **Aaron Fisher** (LLNL) |
-| 3:00-4:00 | **Q&A Session** | **MFEM team** available on Zoom + Slack |
+The meeting activities will take place 8:00am-4:00pm Pacific Daylight Time (GMT-7) on October 22, 23 and 24, with office hours and hackathon in the afternoon of the last day.
+
+Full agenda will be announced on October 1.
```
Comment (pazner) [inline src/workshop.md:62]:
```suggestion
The full agenda will be announced on October 1.
```

File: src/workshop.md [hunk 2]
```diff
@@ -1,46 +1,54 @@
...
-#### Virtual Meeting
-
-**Speakers' slides** are linked in the agenda below. Read the
-[**article**](https://computing.llnl.gov/about/newsroom/mfem-workshop-2023)
-about the workshop on LLNL's Computing website. Watch the
-[**video playlist**](https://www.youtube.com/playlist?list=PLy9rIbGDXrG0crap1EBFBRV_-RrIDa1an) of
-workshop presentations (linked individually below and available on the [videos page](videos.md)),
-and view **contest submissions** in the [gallery](gallery.md).
+#### October 22-24, 2024
+#### LLNL + Virtual
```
Comment (pazner) [inline src/workshop.md:3]: Maybe "In-Person at LLNL + Virtual" or "In-Person at LLNL + Hybrid Virtual Option"

File: src/index.md [hunk 3]
```diff
@@ -92,10 +92,10 @@ See also our [Gallery](gallery.md), [Publications](publications.md), [Videos](vi
 
 Date         | Message
 ------------ | -----------------------------------------------------------------
+Oct 22, 2024 | 2024 [MFEM Community Workshop](workshop.md).
  Jun 5, 2024 | [Register](https://hpcic.llnl.gov/tutorials/2024-hpc-tutorials) for MFEM [tutorial](tutorial/index.md) on Aug 22.
  May 7, 2024 | Version 4.7 [released](https://github.com/mfem/mfem/blob/v4.7/CHANGELOG).
  May 2, 2024 | New MFEM [paper](https://arxiv.org/abs/2402.15940) in IJHPCA.
-Oct 26, 2023 | 2023 [MFEM Community Workshop](workshop.md).
```
Comment (pazner) [inline src/index.md:98]: Do we want to delete this news item complete?

Comment (tzanio) [reply src/index.md:98]: It is still in https://mfem.org/news/

File: src/workshop.md (hunk 2 above)
Comment (tzanio) [reply src/workshop.md:3]: This shows on the side as a menu item so I shortened it not to stand out

Comment (tzanio) [reply src/workshop.md:3]: Changed to "In-person at LLNL + Virtual" which still fits

File: src/workshop.md [hunk 4]
```diff
@@ -139,24 +123,32 @@ Click each image to enlarge, then right-click to save locally.
...
 [*Holly Auten*](https://people.llnl.gov/auten1)
 ┊  [*Aaron Fisher*](https://people.llnl.gov/fisher47)
-┊  [*Milan Holec*](https://scholar.google.cz/citations?user=Y1lrsgYAAAAJ&hl=cs)
 ┊  [*Tzanio Kolev*](https://people.llnl.gov/kolev1)
+┊  [*Justin Laughlin*](https://www.linkedin.com/in/justin-laughlin)
 ┊  [*Ketan Mittal*](https://people.llnl.gov/mittal3)
 ┊  [*Will Pazner*](https://pazner.github.io)
-┊  [*Socratis Petrides*](https://people.llnl.gov/petrides1)
-┊  [*Vladimir Tomov*](https://people.llnl.gov/tomov2)
+┊  [*Sohail Reddy*](https://scholar.google.com/citations?user=trH03DQAAAAJ&hl=en)
```
Comment (sohailreddy) [inline src/workshop.md:144]: We could add Haley too?

Comment (tzanio) [reply src/workshop.md:144]: I'll check with her on Monday

**Pull request merged**